src/components/widgets/ElevationWidget.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import {XYPlotTemplate} from 'components/widgets/shared/XYPlotTemplate';
+import {IDefaultWidgetProps} from 'components/widgets/shared/IDefaultWidgetProps';
+import {IPoint} from 'types/IPoint';
+import {BasicHorizontalAxis, BasicVerticalAxis} from 'components/widgets/shared/BasicAxes';
+import {LinearAreaSeries} from 'components/widgets/shared/BasicAreaSeries';
+
+function timeCompare(a: { x: any; }, b: { x: any; }){
+    let timeA = a.x;
+    let timeB = b.x;
+    let comparison = 0;
+
+    if (timeA > timeB) {
+        comparison = 1;
+    } else if (timeA < timeB) {
+        comparison = -1;
+    }
+    return comparison;
+}
+
+export interface IElevationWidgetProps extends IDefaultWidgetProps {
+  elevation: IPoint[];
+
+  heightPx: number;
```

Comment:
Arent height and width in IDefaultWidgetProps?

Reply:
Done!